incubator/nodejs/image/project/Dockerfile

```diff
@@ -7,11 +7,12 @@ RUN apt-get update \
  && apt-get clean \
  && echo 'Finished installing dependencies'
 
+WORKDIR "/project/user-app/node_modules"
```

Comment:
Are the `nodejs` specific changes in the other PR also?

Reply:
Yes: https://github.com/appsody/stacks/pull/679#issuecomment-589350889